src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -174,18 +276,6 @@ EmbeddedAssemblies::typemap_managed_to_java (const char *managed)
 	return find_entry_in_type_map (managed, mj_typemap, mj_typemap_header);
 }
 
-MONO_API const char *
-monodroid_typemap_java_to_managed (const char *java)
```

Comment:
If you're going to move them *again*, why move them now?

Reply:
Because I dislike how the C wrappers are sprinkled all around the code in the files and while making major changes to this particular file I ran across these two and adding a handful of lines to the diff didn't seem like much trouble, for the sake of a cleaner diff later on at some point. Anyhow, reverted this change.